packages/falso/package.json

```diff
@@ -26,5 +26,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/ngneat/falso"
+  },
+  "dependencies": {
+    "@types/seedrandom": "3.0.1",
+    "seedrandom": "3.0.5"
```

Comment:
now final build includes these dependencies

Reply:
@types/seedrandom should be dev dep.